pymc/tests/test_idata_conversion.py

```diff
@@ -602,6 +602,15 @@ def test_issue_5043_autoconvert_coord_values(self):
             )
             assert isinstance(converter.coords["city"], pd.MultiIndex)
 
+    @pytest.mark.xfail(
+        reason="Xarray doesn't support labeling dimensions with the same name as the variable"
+    )
+    def test_variable_dimension_name_collision(self):
+        with pm.Model() as pmodel:
+            pm.ConstantData("time", np.ones(9), dims="time")
+            pm.Normal("a")
+            idata = pm.sample(chains=1)
```

Comment:
The above two lines are unreachable. You could also call model.register_rv directly

Reply:
Could you be more specific about which 2 lines are unreachable?
I also looked up the documentation of model.register_rv, but I couldn't figure out how to use it.
Would you mind giving me some additional pointers?